src/components/NewGalaxyModal.js

```diff
@@ -111,12 +155,68 @@ export const NewGalaxyModal = _.flow(
     ])
   }
 
+  const renderStoppingWarning = () => {
+    return h(Fragment, [
```

Comment:
nit: A `Fragment` is only needed if there are multiple children, but since there's a single `div` here it's unnecessary.